community/sample-apps/react-apollo-todo/src/apollo.js

```diff
@@ -8,41 +8,44 @@ import { SubscriptionClient } from "subscriptions-transport-ws";
 import { setContext } from "apollo-link-context";
 
 import { GRAPHQL_URL, REALTIME_GRAPHQL_URL } from "./utils/constants";
+import auth from "./components/Auth/Auth";
 
 const getHeaders = () => {
-  const token = localStorage.getItem("auth0:id_token");
   const headers = {
-    authorization: token ? `Bearer ${token}` : ""
+    authorization: `Bearer ${auth.getIdToken()}`
```

Comment:
You changed the ternary check to not return an empty string. Will this be ok if there's no auth header?
Or shouldn't this be:

```javascript
const headers = {};
const token = auth.getIdToken();
if (token) {
headers.authorization = `Bearer ${token}`;
}
return token;
```